people/compensation.md

```diff
@@ -2,17 +2,32 @@
 
 ## Compensation philosophy
 
-See [the 2i2c Careers page](https://2i2c.org/jobs/) for a short description of our compensation philosophy.
-
-Salaries at 2i2c are determined solely by job title as well as several steps within that job (corresponding to experience in a particular job).
-We have [internal salary documents for the specific positions and levels](https://docs.google.com/spreadsheets/d/1FJM5pAbc0EWhu4CpPjlbWTMOsZAnivEd2ZBIZIdwpE8/edit?usp=sharing).
-
 :::{seealso}
 We are [working on an update of our compensation policy](https://docs.google.com/document/d/1Ixkg-YgeTO-Xc9neQHIlTHa7psyhlFHQQCSfCr-K0cQ/edit#).
 See that document for ideas that are more specific / structured than this page.
 :::
 
-### Define compensation levels with GuideStar
+:::{admonition} Aspirational
+We are a young organization, and this page describes the salary policy that we aim to enact in the coming years.
+It may not accurately reflect our current practices, but we share it for transparency and to provide a goal for our team to shoot for.
+:::
+
+Our salary policy should ensure that our team feels financially supported and respected in their roles, and should be aligned with our values and mission as a non-profit organization.
+Team members should feel that 2i2c is a competitive workplace from a salary perspective, while recognizing that we do not have the financial resources of most tech companies.
+Below are a few guiding principles of our salary policy:
+
+- **Benchmarked to non-profit tech**. We should benchmark our salaries based on similar roles in our sector (non-profits that do technical or research work, depending on the role). See [](compensation:salaries:guidestar) for the current process we follow.
+  - If we determine that a sector is systemically under-paying a given role in a way that makes us uncompetitive, then we may increase the salary band for the role to be more aligned with the private sector.
```

Comment:
Thanks for adding this piece. I do think our "guide" should be something else than the guidestar one for the Eng team, and the upcoming proposal I am working on takes that point into account.

Reply:
I also ack Eng "being different" may cause some conflicts that we will need to address as well so other areas of the organization feel OK. My proposal will include a specific point for that as well.